tools/logger/logger.c

```diff
@@ -220,17 +220,15 @@ static void *wait_open(const char *watched_dir, const char *expected_file)
 	const int dwatch = inotify_add_watch(iqueue, watched_dir, IN_CREATE);
 	struct stat expected_stat;
 	void *ret_stream = NULL;
-
-	char * const fpath = malloc(strlen(watched_dir) + 1 + strlen(expected_file) + 1);
+	const int fpath_len = strlen(watched_dir) + 1 + strlen(expected_file) + 1;
```

Comment:
need a static prefix?

Reply:
Why fpath_len variable should be a static?